### PR TITLE
Add tnt test for RouterCallImpl

### PR DESCRIPTION
**Pull request by nurzhan-saktaganov**
* don't run TestConncurrentTopologyChange parallel with other ones
* fix callOpts for RouterCallImpl

### CODE REVIEW

**Comment by KaymeKaydex** [comment]
@osperelygin check it please

File: tests/tnt/router_call_test.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		TopologyProvider: static.NewProvider(cfg),
+		DiscoveryTimeout: 5 * time.Second,
+		DiscoveryMode:    vshardrouter.DiscoveryModeOn,
+		TotalBucketCount: totalBucketCount,
+		User:             defaultTntUser,
+		Password:         defaultTntPassword,
+	})
+
+	if err != nil {
+		panic(err)
```
Comment (KaymeKaydex) [inline tests/tnt/router_call_test.go:38]: lets not panic 
use require.NoError() instead

**Review by KaymeKaydex — CHANGES_REQUESTED** [review]

**Comment by KaymeKaydex** [comment]
and lets add  export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES for MacOS users test safety caused by http://sealiesoftware.com/blog/archive/2017/6/5/Objective-C_and_fork_in_macOS_1013.html

File: tests/tnt/concurrent_topology_test.go [hunk 2]
```diff
@@ -115,7 +116,13 @@ func TestConncurrentTopologyChange(t *testing.T) {
 		return
 	}
 
-	t.Parallel()
+	// suppress the below linter warning:
+	// unused-parameter: parameter 't' seems to be unused, consider removing or renaming it as _ (revive)
+	_ = t
```
Comment (osperelygin) [inline tests/tnt/concurrent_topology_test.go:121]: `t` is passed to the `require.Nil()` method, this line is no longer needed

File: tests/tnt/Makefile [hunk 3]
```diff
@@ -30,16 +31,16 @@ cluster-up:
 		ln -sf `(pwd)`/cfgmaker.lua tmp/$${rsid}/master/cfgmaker.lua;					\
 		ln -sf `(pwd)`/storage.lua tmp/$${rsid}/follower/storage_$${rsid}_follower.lua;	\
 		ln -sf `(pwd)`/cfgmaker.lua tmp/$${rsid}/follower/cfgmaker.lua;					\
-		TT_WORK_DIR=tmp/$${rsid}/master/ TT_PID_FILE="tarantool.pid" TT_BACKGROUND=true START_PORT=${START_PORT} TT_LOG=/dev/null NREPLICASETS=${NREPLICASETS} tarantool tmp/$${rsid}/master/storage_$${rsid}_master.lua;		\
-		TT_WORK_DIR=tmp/$${rsid}/follower/ TT_PID_FILE="tarantool.pid" TT_BACKGROUND=true START_PORT=${START_PORT} TT_LOG=/dev/null NREPLICASETS=${NREPLICASETS} tarantool tmp/$${rsid}/follower/storage_$${rsid}_follower.lua;	\
+		TT_WORK_DIR=tmp/$${rsid}/master/ TT_PID_FILE=tarantool.pid TT_BACKGROUND=true TT_LOG=tarantool.log tarantool tmp/$${rsid}/master/storage_$${rsid}_master.lua;		\
```
Comment (osperelygin) [inline tests/tnt/Makefile:34]: now we save the logs to a file, let's update the comment

https://github.com/KaymeKaydex/go-vshard-router/blob/nsaktganov_tnt_test_coverage_up/tests/tnt/Makefile#L23

File: tests/tnt/router_call_test.go (hunk 1 above)
Comment (nurzhan-saktaganov) [reply tests/tnt/router_call_test.go]: Done

**Comment by nurzhan-saktaganov** [comment]
> one of the problems that we missed is the lack of grant inside storage
> 
> ```
> box.once('access:v1', function()
> 	box.schema.user.grant('guest', 'super')
> end)
> ```

Done

File: tests/tnt/concurrent_topology_test.go (hunk 2 above)
Comment (nurzhan-saktaganov) [reply tests/tnt/concurrent_topology_test.go]: Done

**Comment by nurzhan-saktaganov** [comment]
> and lets add export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES for MacOS users test safety caused by http://sealiesoftware.com/blog/archive/2017/6/5/Objective-C_and_fork_in_macOS_1013.html

Done

**Review by KaymeKaydex — APPROVED** [review]

**Pull request merged**